Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:wily
```

Comment:
There is already a Dockerfile. See https://github.com/giantswarm/mayu/blob/master/Dockerfile.dist